gazupublisher/views/MainWindow_ui.py

```diff
@@ -29,16 +52,26 @@ def setupUi(self, MainWindow):
 
         self.toolbar = CustomToolBar(self)
         self.toolbar.show()
-        self.verticalLayout.addWidget(self.toolbar)
-        self.verticalLayout.addWidget(self.table)
 
-        MainWindow.setCentralWidget(self.centralwidget)
+        self.vertical_layout.addWidget(self.toolbar)
+        self.vertical_layout.addWidget(self.table)
 
-        self.fitToTable()
-        QtCore.QMetaObject.connectSlotsByName(MainWindow)
+    def setup_right(self, task):
```

Comment:
The naming is weird. If you decide to name things from a layout perspective, you should not give a task as a parameter, a widget is expected. You can do the opposite too: rename the function with something like, `setup_task_panel`.